ERCS/erc-7790.md

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
```suggestion
---
eip: 7753
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.